### PR TITLE
Fix task stop and processing task can not be finished

### PR DESCRIPTION
**Pull request by xjlgod**

### CODE REVIEW

File: server/pkg/webserver/service/importer/taskdb.go [hunk 1]
```diff
@@ -81,3 +85,16 @@ func (t *TaskDb) SelectAllIds(nebulaAddress, user string) ([]int, error) {
...
+	var taskInfos []TaskInfo
+	if err := t.Select("id").Where("task_status = ?", StatusProcessing.String()).Find(&taskInfos).Error; err != nil {
+		return err
+	}
+	for _, taskInfo := range taskInfos {
+		if err := t.Model(&taskInfo).Update("task_status", StatusAborted.String()).Error; err != nil {
+			return err
+		}
+	}
+	return nil
```
Comment (veezhang) [inline server/pkg/webserver/service/importer/taskdb.go]: Why not use `update table set f=v where`?

File: server/pkg/webserver/service/importer/taskmgr.go [hunk 2]
```diff
@@ -161,19 +161,22 @@ func (mgr *TaskMgr) UpdateTaskInfo(taskID string) error {
 	StopTask will change the task status to `StatusStoped`,
 	and then call FinishTask
 */
-func (mgr *TaskMgr) StopTask(taskID string) bool {
+func (mgr *TaskMgr) StopTask(taskID string) error {
```
Comment (veezhang) [inline server/pkg/webserver/service/importer/taskmgr.go:164]: Always return error?

**Review by veezhang — COMMENTED** [review]
@xjlgod

File: server/pkg/webserver/service/importer/taskmgr.go (hunk 2 above)
Comment (xjlgod) [reply server/pkg/webserver/service/importer/taskmgr.go:164]: > Always return error?

177 line return nil

**Review by veezhang — APPROVED** [review]
LGTM

**Pull request merged**